src/main/java/duke/note/Note.java

```diff
@@ -0,0 +1,33 @@
+package duke.note;
+
+/**
+ * Note Class.
+ */
+public class Note {
+    private String desc;
+
+    /**
+     * Constructor for Note class.
+     * @param desc description of note
+     */
+    public Note(String desc) {
+        this.desc = desc;
+    }
+
+    /**
+     * Returns the description of a note.
+     * @return description of the note
+     */
+    public String getDesc() {
```

Comment:
What's the difference between the description and string representation of a note?

Reply:
`String` representation is when we are returning duke's response, while getDesc() is to get the description of the note when saving it into the storage. My intention was that although both return the same thing, but since they serve different purposes and will be called under different circumstances, it would help to have different descriptive names describing the context when these methods are being called. However, you have a good point and I might consider changing their names to make them sound suitable for all contexts.